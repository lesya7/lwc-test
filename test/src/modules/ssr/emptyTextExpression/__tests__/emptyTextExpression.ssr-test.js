/*
 * Copyright (c) 2018, salesforce.com, inc.
 * All rights reserved.
 * SPDX-License-Identifier: MIT
 * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
 */
import { generateAndSnapshotMarkup } from '@lwc/ssr-snapshot-utils/src/ssr-snapshot-utils';
import EmptyTextExpression from 'ssr/emptyTextExpression';

it('renders a basic component with an empty text expression', () => {
    const { renderedComponent, snapshotHash } = generateAndSnapshotMarkup(
        'ssr-empty-text-expression',
        EmptyTextExpression,
        {},
    );

    expect(renderedComponent).toMatchSnapshot(snapshotHash);
});
