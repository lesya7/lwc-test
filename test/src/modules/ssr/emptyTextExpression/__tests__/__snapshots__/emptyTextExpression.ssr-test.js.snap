// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a basic component with an empty text expression: 441df09d0ffdded72a83d8c3092542de3ef4e5087d70a631cd883aa0b0fe6a29 1`] = `<ssr-empty-text-expression><template shadowrootmode="open"><div>‚Äç</div></template></ssr-empty-text-expression>`;
